### DISCUSSION TITLE
From old way to EXPO

### DISCUSSION THREAD

**Discussion by emeraldskygroup** [notJust-dev/notjustdev#240] [Q&A] [answered]
Hey Vamim! I hope you're doing well.

Noticed the old tutorials are not using Expo, I tried following but seems everything has changed now that you're using Expo router. Am I still able to follow the old tutorials and implement expo as well?

Thank you!
Marco

**Comment by VadimNotJustDev** [comment, marked as answer]
Yes, everything that's possible in rn-cli is also possible in Expo. There might be situation were you you have to adjust some things, so you need a bit of experience, but I recommend you try to go this route, with expo, and learn along the way. For example, there might be some alternative libraries that would be easier to integrate in expo. A very simple example, if in rn-cli we use react-native-vector-icons, in expo we would use  expo vector-icons, wich is the same library, adjusted to work easier in expo.

**Reply by emeraldskygroup** [reply]
Ah ok! Oh but I was wondering though, if I want to follow your Uber tutorial, you did that one with React CLI, you think I should wait until I feel comfortable and then try it but using Expo? Or you think you will do an Uber tutorial using Expo?
